# The ID of the marker to follow
follow_id: 0


marker_timeout: 1.0

min_ang_vel: 0.1
max_ang_vel: 1.0
angle_min: 0.1
angle_max: 0.7

min_lin_vel_forward: 0.1
max_lin_vel_forward: 0.4
distance_min_forward: 0.7
distance_max_forward: 1.2

min_lin_vel_reverse: 0.1
max_lin_vel_reverse: 0.2
distance_min_reverse: 0.4
distance_max_reverse: 0.5